channels/co.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="Amordiscos.co" tvg-country="CO" tvg-language="Spanish" tvg-logo="https://amordiscostv.com/wp-content/uploads/2021/02/thumbnail_Logo-Amordiscos-600x600-alpha-150x150.png" group-title="Music",Amordiscos
+https://5b464b69d264e.streamlock.net:/Channels_live/ngrp:amordiscos_all/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/135714567-20152f9e-4638-440a-a379-8f3b80d1f8f7.png)

Reply:
![vlcsnap-2021-10-02-16h15m24s071](https://user-images.githubusercontent.com/30985701/135717919-e957e9e0-a83f-4d92-a371-b62a2cc57093.png)

Reply:
Works for me.

Reply:
I've understood why. Let me check.

Reply:
@freearhey Test again

Reply:
now works